release/nightly_tests/dataset/image_loader_microbenchmark.py

```diff
@@ -25,6 +26,219 @@
 FULL_IMAGE_SIZE = (1213, 1546)
 
 
+def benchmark(args):
```

Comment:
This just moves code out of `if __name__ == "__main__"` into a `benchmark` function